test/unit/db/repos/accounts.js

```diff
@@ -67,19 +69,15 @@ function createAccount() {
 
 describe('db', function() {
 	before(function(done) {
-		application.init({ sandbox: { name: 'lisk_test_db_accounts' } }, function(
-			err,
-			scope
-		) {
-			db = scope.db;
+		dbSandbox = new DBSandbox(__testContext.config.db, 'lisk_test_db_accounts');
+
+		dbSandbox.create((err, __db) => {
+			db = __db;
+
 			done(err);
 		});
 	});
 
-	after(function(done) {
-		application.cleanup(done);
```

Comment:
You should call `dbSandbox.destroy();` here.

Reply:
Done.